frontend/src/app/admin/components/admin-content/admin-motif-form/admin-motif-form.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="content-wrapper">
+  <!-- Content Header (Page header) -->
+  <section class="content-header">
+    <h1>
+      Motifs
```

Comment:
Since we treat only the 'Motif de suppression' for now, I would recommend to use here Motifs de Suppression

Reply:
fixed